scripts/templates/selectorsyncset.yaml

```diff
@@ -4,13 +4,8 @@ kind: SelectorSyncSet
 metadata:
   generation: 1
   name: osd-manged-cluster-config
-  labels:
-    managed.openshift.io/osd: True 
```

Comment:
Hive is using all strings. Not sure about reason. We are not (at this time) planning to do anything with these other than help in operations and reporting spits, probably fine.